src/lib/autoinstall/clients/inst_autosetup_upgrade.rb

```diff
@@ -309,7 +309,9 @@ def software_upgrade
         # initialize target
         PackageCallbacks.SetConvertDBCallbacks
 
-        Pkg.TargetInit(Installation.destdir, false)
+        Pkg.TargetInitializeOptions(Installation.destdir,
+          # enable DB rebuild at upgrade (bsc#1209565)
+          "rebuild_db" => true)
```

Comment:
does user has option to also rebuild it during manual upgrade or it is autoupgrade only feature?

Reply:
This is only for autoupgrade, the manual upgrade is handled in https://github.com/yast/yast-update/pull/186